Css/WebScrap.Css/Data/Selectors/AnyChildCssSelector.cs

```diff
@@ -0,0 +1,6 @@
+namespace WebScrap.Css.Data.Selectors;
+
+/// <summary>
+/// Represents ' ' selector.
+/// </summary>
+public record class AnyChildCssSelector : CssSelector;
```

Comment:
I dont like the name. Better rename `>`-selected class for DirectChildSelector,
While this ' '-selected class is just ChildCssSelector.